foreman/data_refinery_foreman/foreman/management/commands/track_data_request.py

```diff
@@ -0,0 +1,133 @@
+from collections import Counter
+
+from django.core.management.base import BaseCommand
+
+import requests
+
+from data_refinery_common.logging import get_and_configure_logger
+from data_refinery_common.models import DownloaderJob, Experiment, ProcessorJob, SurveyJob
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--source-url",
```

Comment:
```suggestion
            "--file",
```

Reply:
This is just to make it consistent with other management commands.

Reply:
Done.